Modules/_sqlite/blob.c

```diff
@@ -347,6 +347,186 @@ blob_exit_impl(pysqlite_Blob *self, PyObject *type, PyObject *val,
     Py_RETURN_FALSE;
 }
 
+static Py_ssize_t
+blob_length(pysqlite_Blob *self)
+{
+    if (!check_blob(self)) {
+        return -1;
+    }
+    return sqlite3_blob_bytes(self->blob);
+};
+
+static int
```

Comment:
```suggestion
static Py_ssize_t
```

Reply:
Should probably also address the other warnings the Windows compiler put in the PR